articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+> [!NOTE]
+> For more information on the benefits of compiling assets and what optimizations it can offer, see the [Content Pipeline](../../../getting_started/content_pipeline/index.md) documentation.
+
+For this tutorial series, we're going to focus on using the content pipeline workflow to loading assets, including images.  Doing this will get you as the developer accustomed to using the content pipeline tools and also give the benefits of having assets precompiled to optimized formats.
+
+To get started, first we need an image to load.  Right-click the following image of the MonoGame logo and save it named *logo.png* somewhere on your on your computer, such as your desktop.
+
+<figure><img src="./images/logo.png" alt="Figure 4-2: MonoGame Horizontal Logo"><figcaption><p><strong>Figure 4-2: MonoGame Horizontal Logo</strong></p></figcaption></figure>
+
+### Adding the Image To Content Project
+Now that we have an image, we need to open the *MonoGame Content Builder Editor (MGCB Editor)* so we can add the image to the content project.  Since we are using VSCode for this tutorial, you should have installed the *MonoGame for VSCode* extension in [Chapter 02](../02_getting_started/index.md#installing-the-monogame-for-vscode-extension).  When this extension is installed, anytime you have a code file open, you will see the MonoGame logo in the top-right of the code window like below:
+
+<figure><img src="./images/mgcb-editor-icon.png" alt="Figure 4-3: MonoGame for VSCode extension icon"><figcaption><p><strong>Figure 4-3: MonoGame for VSCode extension icon</strong></p></figcaption></figure>
+
+Clicking this icon will open the MGCB Editor and load the *Content.mgcb* file from the current project in it.
+
+> [!NOTE]  
+> If you did not install the *MonoGame for VSCode* extension or prefer to not use it, you can use the CLI commands to open the MGCB Editor instead. To do this:
+>
+> 1. Open the terminal in VSCode pressing `` CTRL+` ``or choosing *View > Terminal* from the top menu
+> 2. If the terminal directory is not in the project root directory (the one with the .csproj file), then use the `cd` command to navigate to that directory.
+> 3. Enter the command `dotnet mgcb-editor ./Content/Content.mgcb` to open the MGCB Editor using the *Content.mgcb* file from the project.
+
+<figure><img src="./images/mgcb-editor.png" alt="Figure 4-4: MonoGame Content Builder Editor (MGCB Editor) Window"><figcaption><p><strong>Figure 4-4: MonoGame Content Builder Editor (MGCB Editor) Window</strong></p></figcaption></figure>
+
+With the MGCB Editor now open, perform the following
+
+1. Right-click on the *Content* node in the *Project* panel on the left.
+2. Select *Add > New Folder*.

```

Comment:
Probably worth having a note about folders and paths.  Since some do not realise the relationship when specifying the path and file when loading (came up on Reddit this week)
A highlight section maybe?

Reply:
Added "Understanding Content Paths" section to explain this